packages/constants/src/constants.ts

```diff
@@ -181,7 +181,6 @@ export const MC_API_PROXY_TARGETS = {
   MACHINE_LEARNING: 'ml',
   PIM_SEARCH: 'pim-search',
   ORDER_SEARCH: 'order-search',
-  MC_METRICS: 'mc-metrics',
```

Comment:
Nit: slightly unrelated change. If we'd be strict, then this was a breaking change. Did a quick search across all codebases and it doesn't seem to be used. So I think being pragmatic is fair.

Reply:
Yes I'm fine with a minor/patch version as this is considered more of an internal usage.